docs/Site-Reliability-Guardian/example-pipelines/Github/github.yml

```diff
@@ -0,0 +1,40 @@
+# Demo pipeline for SRG evaluation.
+name: demo-pipeline-srg
+on:
+  workflow_dispatch:
+jobs:
+  test:
+    runs-on: ubuntu-20.04
```

Comment:
Why is it ubuntu-20.04 instead of ubuntu-latest as in the evaluation phase? If there is no difference, it is better to go with a single one

Reply:
good point i will update the value in the next PR with other incremental updates to the docs